### DISCUSSION TITLE
Stellarium script for mutual events of Jupiter's Galilean Moons (2025-2027)

### DISCUSSION THREAD

**Discussion by RobertNufer** [Stellarium/stellarium#3937] [Show and tell]
About every six years the Earth passes the equatorial plane of Jupiter on its journey around the Sun. During this time period mutual events (occultations and eclipses) of the Galilean moons can be observed as seen from the Earth. the next bunch of mutual events will occur between 2025 and 2027:
![Events](https://github.com/user-attachments/assets/e7c4ed15-0e97-42f6-b001-64fa6cfd2b85)

For the 2025-2027 time period I have written a stellarium script that shows 146 of the events from begin to end:
https://robertnufer.ch/06_computing/stellarium/galilean_mutual_events_2025-2027_v20230729.ssc

A short description can be found here:
[stellarium_script_for_mutual_events_of_the_galilean_moons_2025-2027.pdf](https://github.com/user-attachments/files/17277319/stellarium_script_for_mutual_events_of_the_galilean_moons_2025-2027.pdf)

Regards,
Robert Nufer

**Comment by alex-w** [comment]
Very nice script!

I've added link to your cool script onto our website also - https://github.com/Stellarium/stellarium.github.io/commit/b5685f803f5fe1d9bfa66bde37e33b52a1783346 (website: https://stellarium.org/scripts.html)

**Comment by RobertNufer** [comment]
Dear Alex

 

Thank you very much for adding the script to stellarium/scripts.

 

Kind regards

Robert
